src/model/page.urls.ts

```diff
@@ -13,6 +13,10 @@ export const ENTER_INFORMATION: string = SEPARATOR + Templates.ENTER_INFORMATION
 export const DOCUMENT_UPLOAD: string = SEPARATOR + Templates.DOCUMENT_UPLOAD;
 export const DOCUMENT_UPLOAD_CONTINUE: string = SEPARATOR + "document-upload-continue"; // not a template
 export const REMOVE_DOCUMENT: string = SEPARATOR + Templates.REMOVE_DOCUMENT + "/:attachmentId";
+export const DOWNLOAD_ATTACHMENT: string =
+  "/company/:companyId/strike-off-objections/:requestId/attachments/*/download";
```

Comment:
Could be worth doing :attachmentId instead of * so it is clear what the number is, and we can access the field if need be

Reply:
Replaced